idd/Energy+.idd.in

```diff
@@ -81618,6 +81623,58 @@ Matrix:TwoDimension,
   N21001,N21002,N21003,N21004,N21005,N21006,N21007,N21008,N21009,N21010,N21011,N21012,N21013,N21014,N21015,N21016,N21017,N21018,N21019,N21020,  \note fields as indicated
   N21021,N21022,N21023,N21024,N21025,N21026,N21027,N21028,N21029,N21030,N21031,N21032,N21033,N21034,N21035,N21036,N21037,N21038,N21039,N21040;  \note fields as indicated
 
+\ group Hybrid Model
+
+HybridModel:Zone,
+       \memo Zones with measured air temperature data and a range of dates.
+       \memo A range of dates may differ from zones.
+  A1 , \field Name
+       \required-field
+       \type alpha
+  A2 , \field Zone Name
+       \required-field
+       \type object-list
+       \object-list ZoneNames
```

Comment:
@sanghlee-lbnl When I look at the code in HybriModel.cc, a zone list is allowed. I think ZoneAndZoneListNames should be listed instead of ZoneNames as an object-list.